### DISCUSSION TITLE
uh. why is there no ducumentation?

### DISCUSSION THREAD

**Discussion by MansonLuo** [badoo/Reaktive#682] [Q&A] [answered]
i never learn rx before, do i need to learn rxjava
first?

**Comment by CherryPerry** [comment, marked as answer]
Hi, we do not have a dedicated documentation, please refer to [reactivex.io documentation](https://reactivex.io/intro.html). It is not necessary to learn RxJava in particular, but the approach in general. There are other reactive libraries for [other languages](https://reactivex.io/languages.html) too.

**Reply by arkivanov** [reply]
Agree. Also, Reaktive is inspired by RxJava and there are many similarities. But perhaps we could mention in the readme some most important things that are different - e.g. source builders are top-level functions (like `observableOf`, `observableMyEmitter`, `singleFromFunction`, etc.), subject builders, and so on.
